src/test/java/com/rmbcorp/javawriter/JavacTest.java

```diff
@@ -0,0 +1,89 @@
+package com.rmbcorp.javawriter;
+
+import com.rmbcorp.javawriter.autojavac.JavaCompiler;
+import com.rmbcorp.javawriter.clazz.Clazz;
+import com.rmbcorp.javawriter.clazz.ClazzImplManager;
+import com.rmbcorp.javawriter.logman.LoggerManager;
+import com.rmbcorp.javawriter.logman.TempLogger;
+import org.junit.After;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Aspire on 10/29/2016.
+ */
+public class JavacTest {
```

Comment:
append "IT" - use of system resources makes this an integration test more than a unit test